### DISCUSSION TITLE
Unable to see the Network Traffic from other VLan.

### DISCUSSION THREAD

**Discussion by demronan22** [Security-Onion-Solutions/securityonion#1858] [Unsupported Versions] [answered]
Hi Security Onion Team,

Good Day,

We installed Security Onion 2 in VMware. The installation and configuration (by default) is working fine. Also, we were able to see the network traffic from the VLAN Segment where the SO installed.

We would like to inquire why weren't able to see the the network traffic from the other VLAN or entire network even through we created rules to allowed in Core Switch?

Thanks,
Dem

**Comment by dougburks** [comment, marked as answer]
Try running tcpdump on your sniffing interface to see if the traffic is there. If it's not, then you'll need to figure out why your core switch isn't sending it.
